global_body_planner/include/global_body_planner/planning_utils.h

```diff
@@ -231,11 +269,93 @@ void getInterpPlan(const FullState &start_state,
                    const PlannerConfig &planner_config);
 
 // Terrain-based heuristics
-double getZFromState(const State &s, const PlannerConfig &planner_config);
 double getPitchFromState(const State &s, const PlannerConfig &planner_config);
```

Comment:
The get methods can be const?

Reply:
These are just namespaced functions, not in a class.